mlir/lib/Conversion/VectorToLLVM/ConvertVectorToLLVM.cpp

```diff
@@ -1096,6 +1096,26 @@ class VectorExtractOpConversion
     SmallVector<OpFoldResult> positionVec = getMixedValues(
         adaptor.getStaticPosition(), adaptor.getDynamicPosition(), rewriter);
 
+    for (unsigned idx = 0; idx < positionVec.size(); ++idx) {
+      if (auto position = llvm::dyn_cast<Value>(positionVec[idx])) {
```

Comment:
https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code